packages/dev/core/src/FlowGraph/Blocks/Execution/ControlFlow/flowGraphTimerBlock.ts

```diff
@@ -27,13 +22,10 @@ export class FlowGraphTimerBlock extends FlowGraphAsyncExecutionBlock {
      */
     public readonly onTimerDone: FlowGraphSignalConnection;
 
-    constructor(parameters?: IFlowGraphTimerBlockParameters) {
+    constructor() {

```

Comment:
why was the configuration removed?

Reply:
Because without a context on the constructor, we can't set the value on the nodes